proposal-membership.md

```diff
@@ -8,19 +8,29 @@
 
 **Membership criteria**
 1. Pay a monthly membership fee, flexible for each member, with a suggested baseline of 10€ (which can be paid in advance for up to 12 months). See financial stake.
-2. Be actively vouched for by at least 3 active members: A vouch should be initiated by another active member and can later be removed at any time. (see “Membership Split” for limit cases)
+2. Be actively vouched for by at least 3 active members: A vouch should be initiated by another active member and can later be removed at any time. (see below: "Vouching" for the process, and “Membership Split” for limit cases)
 3. Have physical or legal presence in Europe, or be involved in an active consortium project with at least one member with a physical or legal presence in Europe
 4. Be a natural person.
 5. If not a member for the last 12 months:
     1. Publicly introduced themselves to the existing members (ex: background, motivation, skills, etc.)
-    2. Have made recognized contributions to the larger SSB community (ex: tools, artwork, community gardening, volunteering, etc.)
 6. If a member for the last 12 months: 
     1. Voted in the last annual general assembly
     2. Have volunteered in the last 12 months on tasks required to keep the consortium running and reported those contributions
     3. Have attended at least 1 of the last 3 physical gatherings.
 
 If a member fails to maintain their active status by omitting to fulfill some of the criteria above, they become an “Inactive Member” for up to a year, and lose voting rights. An active member will contact them to understand why. The “Inactive Member” can then choose to: (1) fulfill the criteria to become “Active” again, or (2) become a “Past Member”. In the absence of answer, after one year of “Inactive” status, they automatically become “Past Members”. (see “Recognizing Past Members” below)
 
+After fulfilling the criteria above, new members are added to a Signal group representing the collective. If members fail to maintain their active status, they are removed from the Signal group at the moment they become Inactive.
+
+### Vouching
+
+When you vouch for another member, it means: "I trust this person and their abilities to participate fruitfully in the collective. I take on the responsibility of introducing the rules and spirit of the collective, and agree to intermediate if they are part of or cause conflicts."
+
+Vouchings are public within the collective but private from non-members. For now, vouching relies on messages on Signal and your memory. Your responsibility in vouching also includes knowing who else vouches for the same person. Participate in the following ways:
+  1. **Add a vouch**: publicly message the other members of the Signal group mentioning who you are vouching for. This is also a good time to celebrate and publicly express the good things about your new vouchee!
```

Comment:
why "publicly"? Didn't the paragraph above say that vouching is "private from non-members"?

Reply:
I think it's clear enough from the context that "publicly express etc" means publicly within the consortium

Reply:
Still confusing IMO